File: generated-sources/dart-jaguar/mojang-sessions/lib/model/session_api_error.dart
import 'package:jaguar_serializer/jaguar_serializer.dart';

part 'session_api_error.jser.dart';

class SessionApiError {
   /* A message describing the error */
  @Alias('error')
  final String error;
   /* The request path */
  @Alias('path')
  final String path;
  

  SessionApiError(
    

{
     this.error = null,  
     this.path = null 
    
    }
  );

  @override
  String toString() {
    return 'SessionApiError[error=$error, path=$path, ]';
  }
}

@GenSerializer()
class SessionApiErrorSerializer extends Serializer<SessionApiError> with _$SessionApiErrorSerializer {

}

File: generated-sources/scala-gatling/mojang-sessions/src/gatling/scala/org/openapitools/client/model/SessionApiError.scala

package org.openapitools.client.model


case class SessionApiError (
    /* A message describing the error */
    _error: Option[String],
    /* The request path */
    _path: Option[String]
)
object SessionApiError {
    def toStringBody(var_error: Object, var_path: Object) =
        s"""
        | {
        | "error":$var_error,"path":$var_path
        | }
        """.stripMargin
}
